packages/core/src/Stores/notification-store.js

```diff
@@ -603,6 +611,52 @@ export default class NotificationStore extends BaseStore {
         const platform_name_go = getPlatformSettings('go').name;
 
         const notifications = {
+            acuity: {
+                key: 'acuity',
+                header: localize('New trading tools for MT5'),
+                message: localize('Power up your Financial trades with intuitive tools from Acuity.'),
+                secondary_btn: {
+                    text: localize('Learn More'),
+                    onClick: () => {
+                        ui.setIsAcuityModalOpen(true);
+                        this.removeNotificationByKey({ key: this.client_notifications.acuity.key });
+                        this.removeNotificationMessage({
+                            key: this.client_notifications.acuity.key,
+                            should_show_again: false,
+                        });
+                    },
+                },
+                platform: [platform_name.DTrader],
+                is_disposable: true,
+                img_src: getUrlBase('/public/images/common/acuity_banner.png'),
+                img_alt: 'Acuity',
+                className: 'acuity',
+                type: 'news',
+            },
+            acuity_mt5_download: {
+                key: 'acuity_mt5_download',
+                header: localize('Power up your trades with Acuity'),
+                message: localize(
+                    'Download intuitive trading tools to keep track of market events. The Acuity suite is only available for Windows, and is most recommended for financial assets.'
+                ),
+                secondary_btn: {
+                    text: localize('Learn More'),
+                    onClick: () => {
+                        ui.setIsAcuityModalOpen(true);
+                        this.removeNotificationByKey({ key: this.client_notifications.acuity_mt5_download.key });
+                        this.removeNotificationMessage({
+                            key: this.client_notifications.acuity_mt5_download.key,
+                            should_show_again: false,
+                        });
+                    },
+                },
+                platform: [platform_name.DMT5],
+                img_src: getUrlBase('/public/images/common/acuity_software.png'),
+                img_alt: 'Acuity Download',
```

Comment:
Should we also localize this for better accessibility?

Reply:
resolved : )